doc/boot_arguments.md

```diff
@@ -15,6 +15,29 @@ See [URL handling in the
 installer](https://github.com/yast/yast-installation/blob/master/doc/url.md) to find more details
 about the supported URLs.
 
+## Custom Installation URL Configuration
+
+You can override the default `installation_url` set in the product files [here](https://github.com/openSUSE/agama/tree/master/products.d) by passing the `agama.install_url` parameter as a boot option in the bootloader.
+This is particularly useful for any pre-production testing in openQA. 
+
+**Note:** Setting this variable will impact all products. You can specify multiple URLs by separating them with commas.
+
+### Example Usage
+
+To specify a custom installation URL, pass following as a parameter to kernel in the bootloader:
+
+```
+agama.install_url=https://myrepo,https://myrepo2
+```
+
+### Multiple URLs
+
+
+### Example Usage
+
+To specify a custom installation URL, use the following syntax:
+
```

Comment:
here it missing how syntax with multiple repositories will look like.

Reply:
Will fix.